backend/examples/summa_solvency_flow.rs

```diff
@@ -9,35 +9,37 @@ use summa_backend::{
         address_ownership::AddressOwnership,
         round::{MstInclusionProof, Round},
     },
-    contracts::signer::AddressInput,
+    contracts::signer::{AddressInput, SummaSigner},
+    sample_entries::*,
     tests::initialize_test_env,
 };
-use summa_solvency::merkle_sum_tree::utils::generate_leaf_hash;
+use summa_solvency::merkle_sum_tree::{utils::generate_leaf_hash, MerkleSumTree};
 
 const N_ASSETS: usize = 2;
 const USER_INDEX: usize = 0;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
     // Initialize test environment without `address_ownership` instance from `initialize_test_env` function.
-    let (anvil, _, _, _, summa_contract) = initialize_test_env().await;
+    let (anvil, _, _, _, summa_contract) = initialize_test_env(None).await;
 
     // 1. Submit ownership proof
     //
-    // Each CEX prepares its own `signature` CSV file.
-    let signature_csv_path = "src/apis/csv/signatures.csv";
-
+    // The signer instance would be shared with `address_ownership` and `round` instances
+    //
     // Using AddressInput::Address to directly provide the summa_contract's address.
     // For deployed contracts, if the address is stored in a config file,
     // you can alternatively use AddressInput::Path to specify the file's path.
```

Comment:
Do we have an example of such config file? Can you add it to the comments if possible?

Reply:
Yes, there is "backend/src/contracts/deployments.json".
I will add comment

Reply:
Updated [c575a29](https://github.com/summa-dev/summa-solvency/pull/179/commits/c575a299d5a585ea43204508a74541eda53af37e)